test/BackupClient.test.ts

```diff
@@ -1,16 +1,15 @@
-import assert from 'assert'
 import { BackupType } from '../src/message-backup/BackupClient'
 import { newLocalHostClient, newDevClient } from './helpers'
 
 describe('Backup configuration', () => {
   it('Uses XMTP backup for localhost', async function () {
     const c = await newLocalHostClient()
-    assert.equal(c.backupType, BackupType.xmtpTopicStore)
```

Comment:
Good riddance to these. I think our use of `assert` actually predates my time at XMTP